plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -266,7 +268,13 @@ protected void copyTableSchema(Connection connection, String catalogName, String
                 return Optional.of(dateColumnMapping());
 
             case Types.TIME:
-                return Optional.of(timeColumnMapping(TIME));
+                // TODO support time(p)
```

Comment:
Isn't this PR about adding support to read time(p)? What's missing?

Reply:
leftover, thanks